jujubundlelib/changeset.py

```diff
@@ -73,6 +73,16 @@ def handle_services(changeset):
             'requires': [charms[service['charm']]],
         })
         changeset.services_added[service_name] = record_id
+        if 'annotations' in service:
+            changeset.send({
+                'id': 'setAnnotations-{}'.format(changeset.next_action()),
+                'method': 'setAnnotations',
+                'args': [
+                    service['annotations'],
```

Comment:
In the previous methods we have matched the `args` array to match the env method in the GUI. In this case the updateAnnotations method has the following signature `update_annotations: function(entity, type, data, callback) {` so a more appropriate `args` structure here might be:

```Python
args: [
  record_id,
  'service',
  service['annotations']
]
```